.gitignore

```diff
@@ -8,3 +8,8 @@ www/
 *.log
 *.tmp
 *.tmp.*
+# do not commit macOS specific files
+**/.DS_Store
+# do not commit generated images
+resources/**/icon/
```

Comment:
The generated (by `ionic cordova resources`, I guess) files are committed on purpose. The Ionic online service has been quite unreliable in the past (and is subject to change in the future - I can look up more specific information if you want), and I want users to be able to build this without having to create an Ionic account.